From 261a22096c79f8e6ef7b387514a74d208e4e5945 Mon Sep 17 00:00:00 2001
From: Li xin <lixin.fnst@cn.fujitsu.com>
Date: Fri, 21 Aug 2015 18:23:13 +0900
Subject: [PATCH] config_os_headers: Error Fix

ERROR: This autoconf log indicates errors, it looked at host include
and/or library paths while determining system capabilities.
cc1: warning: include location "/usr/local/include" is unsafe for cross-compilation [-Wpoison-system-directories]
conftest.c:168:17: fatal error: pkg.h: No such file or directory
 #include <pkg.h>
                 ^

Upstream-Status: pending

Signed-off-by: Li Xin <lixin.fnst@cn.fujitsu.com>
---
 configure.d/config_os_headers | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

Index: net-snmp-5.7.3/configure.d/config_os_headers
===================================================================
--- net-snmp-5.7.3.orig/configure.d/config_os_headers
+++ net-snmp-5.7.3/configure.d/config_os_headers
@@ -484,8 +484,8 @@ then
     unset ac_cv_header_pkg_h
     netsnmp_save_CPPFLAGS="$CPPFLAGS"
     netsnmp_save_LDFLAGS="$LDFLAGS"
-    CPPFLAGS="$CPPFLAGS -I/usr/local/include"
-    LDFLAGS="$LDFLAGS -L/usr/local/lib"
+    CPPFLAGS="$CPPFLAGS"
+    LDFLAGS="$LDFLAGS"
     AC_CHECK_HEADERS(pkg.h,
         NETSNMP_SEARCH_LIBS(pkg_init, pkg,
 	    AC_DEFINE(HAVE_LIBPKG, 1, [define if you have BSD pkg-ng])))
